module/combat-tracker/quick-roll-menu.js

```diff
@@ -4,7 +4,8 @@ import { TokenActions } from '../token-actions.js'
 export const addQuickRollButton = async (html, combatant, token) => {
   const quickRollSettings = game.settings.get(Settings.SYSTEM_NAME, Settings.SETTING_USE_QUICK_ROLLS)
   const canShowButtons = quickRollSettings.enabled && (game.user.isGM || combatant.isOwner)
-  if (!canShowButtons || !token.actor) return html
+
```

Comment:
I was glad to see this -- I'm a big proponent of using whitespace/blank lines to make code more readable!